### DISCUSSION TITLE
Code generation from templates. Flask App

### DISCUSSION THREAD

**Discussion by cekisakurek** [pointfreeco/swift-composable-architecture#2591] [Show and tell]
Hello TCA community,
I have open sourced a tool I created where you can generate code from templates. It is a python flask-app and you can access it via https://codegena.cihan.rocks/ or GitHub link is https://github.com/cekware/codegena 

I hope it helps and I welcome all PRs and any feedback.
Thank you!

**Comment by mbrandonw** [comment]
Hi @cekisakurek, thanks for sharing, but it doesn't seem this tool is TCA specific and so I don't think it's really appropriate for the library's discussions. I am going to close it for now.

**Reply by cekisakurek** [reply]
Sorry I thought I can post it on "Show and tell" section.

**Reply by mbrandonw** [reply]
No worries, we just want to keep discussions to TCA-specific topics for right now. If you have a concrete example of using the code generation for TCA, then that might be more appropriate.
